query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/CreateMutationSpec.scala

```diff
@@ -112,15 +112,16 @@ class CreateMutationSpec extends FlatSpec with Matchers with ApiSpecBase {
     server.queryThatMustFail(
       s"""mutation {createScalarModel(data: {optString: "test", optInt: B, optFloat: 1.234, optBoolean: true, optEnum: A, optDateTime: "2016-07-31T23:59:01.000Z" }){optString, optInt, optFloat, optBoolean, optEnum, optDateTime }}""",
       project = project,
-      0,
-      errorContains = """Value types mismatch. Have: Enum("B"), want: Scalar(Int)"""
+      errorCode = 0,
+      errorContains = """ObjectValidationError { object_name: \"ScalarModelCreateInput\", inner: FieldValidationError { field_name: \"optInt\", inner: ValueTypeMismatchError { have: Enum(\"B\"), want: Scalar(Int) }"""
+       // """Value types mismatch. Have: Enum("B"), want: Scalar(Int)"""
```

Comment:
These should be pretty easy to make nicer

Reply:
:+1